.github/workflows/handover-translations.yml

```diff
@@ -0,0 +1,37 @@
+name: Hand over tranlation files
+
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - 'messages/**'
+      - '!messages/GE.json'
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set current date as env variable
+      run: echo "today=$(date +%Y%m%d%H%M)" >> $GITHUB_ENV
+
+    - name: Create folder
+      run: |
+        mkdir -p messages/translated-content && cp messages/GE_* messages/translated-content/
+        ls messages/translated-content
+
+    - name: Create pull request to handover translations
+      uses: paygoc6/action-pull-request-another-repo@v1.0.1
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+      with:
+        source_folder: 'messages/translated-content/**'
+        destination_repo: 'microsoftgraph/microsoft-graph-explorer-v4'
+        destination_folder: 'src/messages'
+        destination_base_branch: 'dev'
+        destination_head_branch: handover-translations/${{ env.today }}
```

Comment:
This is a curious question: Why do you need to create differently named branches, named by dates? From my experience with the permissions-scraper (content-assistant) bot, new incoming commits will just get added to an existing PR if it is still unmerged.

Reply:
This makes sense.. There were no strong factors to consider when naming of the branch.